recipes/napari-live-recording/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-live-recording" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari_live_recording-{{ version }}.tar.gz
+  sha256: 04c367550953ffe60949e53c0c02d436660aa523047b9c92d833b78cf096ae53
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.10
+    - setuptools-scm
+  run:
+    - dask-image
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - opencv
+    - pyqt
+    - python >=3.6,<3.10
```

Comment:
Any specific reason why 3.10 is not compatible? We usually leave open upper bounds. Same deprecation issue @goanpeca?

Reply:
Yes, the same deprecation issue on a dependency of this package `from collection import Iterable` (or similar) that is no longer valid in 3.10.